src/pages/home/home.ts

```diff
@@ -35,4 +36,14 @@ export class HomePage {
       this.tag = '';
     }
   }
+
+  onScan() {
+    BarcodeScanner.scan().then(
+      barcodeData => {
+        this.tag = barcodeData.text;
+        this.onNext();
+      },
+      err => console.error('scan failed: ' + err)
```

Comment:
We should implement better error handling in the future. Created an issue for this: #69.